.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: nix
+nix: 2.1.1
```

Comment:
Why not something more recent?

Reply:
I think we're running 2.2.2 in 19.03

Reply:
because copy/paste from morph :P yea lets update

Reply:
in both projects